backend/api/publishing.py

```
@@ -117,43 +344,168 @@ async def save_scenario_draft(
+                    learning_outcomes_completed=False,
+                    ai_enhancement_completed=False,
+                    grading_config_completed=False,
+                    created_at=datetime.utcnow(),
+                    updated_at=datetime.utcnow()
+                )
+                try:
+                    db.add(scenario)
+                    db.flush()
+                except Exception as e:
+                    if "unique_title_per_user_active" in str(e) or "unique_title_per_user" in str(e):
+                        debug_log(f"Unique constraint violation - scenario with same title already exists, updating instead")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)